feature-libs/customer-ticketing/components/list/customer-ticketing-list/customer-ticketing-list.component.html

```diff
@@ -0,0 +1,179 @@
+<ng-container *ngIf="tickets$ | async as ticketList; else spinner">
+  <span class="cx-ticketing-list-title">
+    <h2 class="cx-ticketing-list-title-text">
+      {{ 'customerTicketingList.requestTitle' | cxTranslate }}
+      ({{ ticketList.pagination?.totalResults || 0 }})
+    </h2>
+  </span>
+  <ng-container
+    *ngIf="
+      ticketList.tickets && ticketList.tickets.length > 0;
```

Comment:
You can make use of optional chaining for catching undefined and null.

```suggestion
    ticketList?.tickets?.length > 0;
```